scripts/publish.ts

```diff
@@ -1,12 +1,12 @@
 import fs from 'fs';
 import path from 'path';
 
-import npmPublish from '@jsdevtools/npm-publish';
+import { npmPublish } from '@jsdevtools/npm-publish';
 import { Results } from '@jsdevtools/npm-publish/lib/results';
 
 const tag = 'dev';
-const dryRun = false;
-const token = process.env.NPM_TOKEN;
+const dryRun = process.argv.includes('--dry-run');
+const token = process.env.NPM_TOKEN ?? 'invalid_token';
```

Comment:
FYI, `@jsdevtools/npm-publish` requires the `token` variable to be a non-empty string. 

And another interesting point: you can run this script with the `--dry-run` flag even if the token is invalid.